pkg/kf/commands/autoscaling/create_autoscaling_rule.go

```diff
@@ -87,8 +87,10 @@ func NewCreateAutoscalingRule(
 				return fmt.Errorf("failed to add autoscaling rule for App: %s", err)
 			}
 
+			app, _ := client.Get(cmd.Context(), p.Space, appName)
+			stopped := app == nil || app.Spec.Instances.Stopped
```

Comment:
I think there's an additional check that can be done here about whether autoscaling is enabled or not, same for update autoscaling limits

Reply:
Would that look something like 

```
stopped := app != nil && (app.Spec.Instances.Stopped || !app.Spec.Instances.Autoscaling.Enabled)
```

i.e. we only really want to wait if the instance is running and autoscaling is enabled?

Reply:
yep, I think that would do it; probably no need for the nil check once you're getting the app back from the transformer though and double check that nothing in the path is nullable (based on the enable autoscaling command it seems like it's not)